recipes/link_traits/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: link-traits
+  version: "1.0.1"
+
+source:
+  fn: link_traits-1.0.1.tar.gz
+  url: https://pypi.python.org/packages/64/d1/4d683994fbba2d3691f7433c09453ea8f4a5144a74767bd39c0d0f8ad25a/link_traits-1.0.1.tar.gz
+  md5: 7f270bc64ccd606573e9dd1d929f0f63
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - traits
+
+  run:
+    - python
+    - traits
+
+test:
+  imports:
+    - link_traits
+
+  commands:
+    - py.test --pyargs link_traits
+
+  requires:
+    - pytest
+    - traitlets
+
+about:
+  home: https://github.com/hyperspy/link_traits
+  license: BSD
+  summary: "A fork to traitlets' link and dlink to link traits in addition to traitlets."
+  license_family: BSD
```

Comment:
Please add the license to the `MANIFEST.in` in the upstream source repo and temporarily package the license with the recipe as in:

http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually